packages/react/src/components/Button/Button.js

```diff
@@ -124,7 +124,7 @@ const Button = React.forwardRef(function Button(
     return () => document.removeEventListener('keydown', handleEscKeyDown);
   }, []);
 
-  const enabled = useFeatureFlag('enable-v11-release');
+  const enabled = FeatureFlags.enabled('enable-v11-release');
```

Comment:
Something that may be different with the hook versus the top-level import is that the hook taps into a "feature flag scope". 

In other words, there is a global set of feature flags that are either turned on/off. When in React and using hooks, we create a specific group of feature flags that could be different than this global set. This is called a "feature flag scope" in our package: https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/FeatureFlags/index.js#L35-L39

This allows different parts of an app to render with different flags (for example, a page could use the v11 flag while all the others do not).

This whole thing gets a little complicated because our props are flagged, but in general we've been using the hook to try and allow for this use-case.

I hope that was helpful, wanted to share more about why the hook is used versus the top-level important and what is different between them 👀